Detail/Extensions/UIFont+Detail.swift

```diff
@@ -7,43 +7,43 @@ import UIKit
 public extension UIFont {
 
     public static var title1: UIFont {
-        let font = UIFont(name: "FINNTypeWebStrippet-Medium", size: 36.0)!
+        let font = UIFont(name: "FINNTypeWebStrippet-Medium", size: 36) ?? UIFont.systemFont(ofSize: 36, weight: .medium)
```

Comment:
Shouldn't we remove the FINN fonts all together then?

Reply:
I don't think we have to, when we import this library it will use the FINN fonts in our app, but if your app doesn't have the fonts it will default to Helvetica Neue.